src/hooks/useGridKeyboardNavigation/__stories__/useGridKeyboardNavigation.stories.mdx

```diff
@@ -0,0 +1,131 @@
+import { useRef, useCallback, useState, useMemo } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import useGridKeyboardNavigation from "../useGridKeyboardNavigation";
+import { action } from "@storybook/addon-actions";
+import cx from "classnames";
+import range from "lodash/range";
+import "./useGridKeyboardNavigation.stories.scss";
+import Button from "../../../../src/components/Button/Button";
+
+<Meta title="Hooks/useGridKeyboardNavigation" />
+
+# useGridKeyboardNavigation
+
+- [Overview](#overview)
+- [Usage](#usage)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Used for accessible keyboard navigation. Useful for components rendering items that can be navigated and selected with a keyboard.
+
+
+
+export const ELEMENT_WIDTH_PX = 72;
+export const PADDING_PX = 24;
+
+export const ON_CLICK = action('item selected');
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+    const ref = useRef(null);
+    const [itemsCount, setItemsCount] = useState(15);
+    const [numberOfItemsInLine, setNumberOfItemsInLine] = useState(4);
+    const width = useMemo(() => numberOfItemsInLine * ELEMENT_WIDTH_PX + 2 * PADDING_PX, [numberOfItemsInLine]);
+    const items = useMemo(() => range(itemsCount).map(num => `${num}.`), [itemsCount]);
+    const getItemByIndex = useCallback(index => items[index], [items]);
+    const { activeIndex, onSelectionAction } = useGridKeyboardNavigation({
+      ref,
+      numberOfItemsInLine,
+      itemsCount,
+      getItemByIndex,
+      onItemClicked: ON_CLICK,
```

Comment:
When developer will copy this code ON_CLICK will not be defined for him.
Form what I understood from @hadasfa presentations story should work just after copy/paste.
Maybe we need to consider to change to something like:
```suggestion
      onItemClicked: () = > {console.log("item selected")},
```

Reply:
that's a good point. the thing is that the `ON_CLICK` actually has some functionality (it adds an action to the "Actions" tab of the story). I also assume that the developer will have to change the callback either way (I guess they won't like a console log as a callback 😃 ).
@hadasfa what do you think about this?

Reply:
I'm gonna merge it as is for now, but if needed I'd be happy to change it in the future :)